code/modules/crafting/quality_of_crafting/crafting.dm

```diff
@@ -481,6 +481,23 @@
 	output = /obj/item/clothing/neck/psycross
 	craft_time = 5 SECONDS
 
+/datum/repeatable_crafting_recipe/crafting/bottle_kit
+	name = "bottle kit"
+	requirements = list(
+		/obj/item/reagent_containers/glass/bottle = 1,
+		/obj/item/paper = 2,
```

Comment:
doesmt this conflict with the bottle labeling mechanic ? 
also should prob be a little more expensive . maybe 2 bottle or 3 . since from what i know the bottling kit can be used multiple times.

may also be interesting to add it to the goldface as a tool.